testsuite/forge-cli/src/main.rs

```diff
@@ -404,7 +408,7 @@ fn land_blocking_test_suite() -> ForgeConfig<'static> {
 fn pre_release_suite() -> ForgeConfig<'static> {
     ForgeConfig::default()
         .with_initial_validator_count(NonZeroUsize::new(30).unwrap())
-        .with_network_tests(&[&NetworkChaosTest])
+        .with_network_tests(&[&NetworkPartitionTest])
```

Comment:
did you mean to just run the partition test here?

Reply:
Actually, I meant to run the bandwidth test for now.  We need to work on some improvements on the transaction emitter side to do that. But we can go ahead with enabling the bandwidth test for nightly tests for now.